tuned-main.conf

```diff
@@ -49,3 +49,23 @@ log_file_max_size = 1MB
 # It can be used to force tuning for specific architecture.
 # If commented, "/proc/cpuinfo" will be read to fill its content.
 # cpuinfo_string = Intel
+
+# Enable TuneD listening on dbus
+# enable_dbus = 1
+
+# Enable TuneD listening on unix domain socket
+# enable_unix_socket = 1
+
+# Path to socket for TuneD to listen
+# Existing files on given path will be removed
+# unix_socket_path = /run/tuned/tuned.sock
+
+# Paths to sockets for TuneD to send signals to separated by , or ;
+# unix_socket_signal_paths =
+
+# default unix socket ownership
+# (uid and gid, python2 does not support names out of box, -1 leaves default)
```

Comment:
While I applaud the effort to go the extra mile to keep things compatible, we should really consider dropping support for python2 if/when that holds us back.

Reply:
This is above "my pay grade". @yarda ?

Reply:
We will drop python 2 in the future, but currently we support RHEL-7, thus we shouldn't break it.  Once RHEL-7 will go EOL we could probably drop it.